### ISSUE TITLE
Use adaptive learning rate for dense_embedding.py

### ISSUE THREAD

**Issue by alessiosavi** [alessiosavi/tensorflow-race-recognition#2]
Instead of use constant learning rate, use adaptive in : https://github.com/alessiosavi/tensorflow-face-recognition/blob/62626eba92eed7fb21f4354cf38fa66c6240fb83/dense_embedding.py#L129

Use something like:
```python
tf.keras.callbacks.ReduceLROnPlateau(
    monitor='val_loss', factor=0.1, patience=10, verbose=0,
    mode='auto', min_delta=0.0001, cooldown=0, min_lr=0, **kwargs
)
```
